.github/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## [v0.7] - 2020-02-28
+
+* Eager mode: Fix parallel RU bug (#57)
 
```

Comment:
You might want to add bugfix #55

Reply:
yes, indeed... gp, should have checked for anything else that made it into `master` since `0.6` as well